docs/orphaned_and_conflicting.md

```diff
@@ -0,0 +1,82 @@
+
+### Deal with "first gen" files
+
+If your locale already contains a `conflicting` or `orphaned` directory, it has been populated by earlier sync commits. For each page, you will need to decide a course of action. Afterwards, rinse and repeat for the other pages.
+
+Depending on your team and on the directory you are processing, you may either go with a pull request dealing with only on page or with a PR for a small set of pages (e.g. processing the whole `conflicting/web/svg` directory at once if it contains 5-10 pages).
+
+#### Orphaned pages
+
+For an orphaned page, the generic approach consists of the following:
+
+1. Identify the `mdn/content` commit for the deletion using:
+   ```bash
+   git log -- files/en-us/slug/to/document.md
+   ```
+   Then check the corresponding PR to have a better understanding of the change.
+2. Check if there is an active redirect for the corresponding page in English (see https://github.com/mdn/content/blob/main/files/en-us/_redirects.txt)
+3. Depending on the presence of a redirect, use either
+    ```bash
+    yarn content delete <orphaned/slug/of/page> <locale> --redirect <other/slug>
+    ```
+
+    or
+
+    ```bash
+    yarn content delete <orphaned/slug/of/page> <locale>
+    ```
+
+#### Conflicting pages
+
+A conflicting page might need more work as content may have been moved/rewritten on the target page as well and redirection might not suffice. That being written, dealing with a conflicting page usually involves:
+
+1. Identify the `mdn/content` commit for the move using:
+   ```bash
+   git log -- files/en-us/slug/to/document.md
+   ```
+   Then check the corresponding PR to have a better understanding of the change.
+2. Applying the same redirect as per `mdn/content` for the source page:
+    ```bash
+    yarn content delete <conflicting/slug/of/page> <locale> --redirect <other/slug>
+    ```
+3. Updating the target page to reflect the movement of content in English. You might need to reuse content from the "conflicting" page into the existing target page to follow the English evolution.
+
+### Deal with generated PRs
+
+With the automated PRs workflow, it is now easier to "catch" orphaned or conflicting pages before they fill the stack of pages to be removed in the repository. 
+
+The process stays the same as above (using `yarn content delete bla/bla locale --redirect foo/foo` most of the time). However, the changes should be made on the branch created by the bot rather than on a new PR (checkout the existing branch rather than creating your own on your fork).
+
+Be careful though, if left opened, the PR will be force-pushed updated by the bot every 24h and your changes might be lost. Be sure to keep a local/forked branch with your changes or to review such changes swiftly within your team.
+
+### "Nice to have"s
```

Comment:
To me this seems to be a typo but not pretty sure, just wanted to double check with you.

```suggestion
### "Nice to have's"
```